src/connector/src/source/cdc/mod.rs

```diff
@@ -156,6 +159,13 @@ where
     }
 }
 
+impl<T: CdcSourceTypeTrait> crate::source::UnknownFields for CdcProperties<T> {
+    fn unknown_fields(&self) -> HashMap<String, String> {
+        // FIXME: CDC does not handle unknown fields yet
+        HashMap::new()
+    }
```

Comment:
when is this func called?

Reply:
<img width="952" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/37948597/9d4a164d-011f-4860-a64f-692db1ecf799">

Reply:
Oh,just missed it. The solution LGTM.